fields/types/text/TextColumn.js

```diff
@@ -17,7 +17,7 @@ var TextColumn = React.createClass({
 	render () {
 		const value = this.getValue();
 		const empty = !value && this.props.linkTo ? true : false;
-		const className = this.props.col.field.monospace ? 'ItemList__value--monospace' : undefined;
+		const className = this.props.col.field && this.props.col.field.monospace ? 'ItemList__value--monospace' : undefined;
```

Comment:
If we're having crashes here, it means something lower level has gone horribly wrong - this check shouldn't be necessary, we should be validating it higher up